### DISCUSSION TITLE
Converting Muti-Page TIFF file into a Multi-page JPG

### DISCUSSION THREAD

**Discussion by Schievelbein** [ImageMagick/ImageMagick#3073] [Help]
I scan a lot of Genealogy photos.
I use TIFF to preserve the images as much as possible.
I scan the front AND back of most photos to get the information on the back.
When I share the photos, many people do not like TIFF and I need to convert them to JPGS.
If I use "Convert Test.tiff Test.jpg" I get Test-0.jpg and Test-1.jpg and then I have to use the Append to bring them back together. with 0 on top and 1 on bottom.  Is there any way to do this with ONE COMMAND?  
BTW, I am using linux, but should not matter.

**Comment by fmw42** [comment]
Try

`convert image.tiff -append image.jpg`

**Reply by Schievelbein** [reply]
THANK YOU very much.
